### ISSUE TITLE
Entropy Detector return NAN- Values

### ISSUE THREAD

**Issue by tla93** [kkirchheim/pytorch-ood#84]
The Entropy Detector occasionally returns NaN values because the softmax function sometimes outputs 0.0, leading to` log(0.0)`, which is undefined and results in a NaN.

**Comment by kkirchheim** [comment]
Fixed by https://github.com/kkirchheim/pytorch-ood/pull/85
